paasta_tools/contrib/create_paasta_playground.py

```diff
@@ -0,0 +1,57 @@
+import json
+import os
+
+from render_template import render_values
+
+
+def main():
+    cluster = os.environ.get(
+        "PAASTA_TEST_CLUSTER", "kind-{USER}-k8s-test".format(**os.environ)
+    )
+    config_path = "etc_paasta_playground"
+
+    # create an etc_paasta_playground directory if it doesn't exist
+    # and copy fake_etc_paasta content into etc_paasta_playground directory
+    if not os.path.isdir("etc_paasta_playground"):
+        os.mkdir("etc_paasta_playground")
+
+    render_values(
+        "./k8s_itests/deployments/paasta/fake_etc_paasta", "etc_paasta_playground", None
```

Comment:
nit: we define `config_path` above, but don't use it here even though we're referring the same thing

Reply:
also: we don't need to do this now but it'd be nice to separate the data we use for integration tests from the data we use to spin up a playground at some point